advanced03-AF_XDP/README.org

```diff
@@ -112,7 +115,7 @@ allocating memory and copying over the frame.
 The end goal of this lesson is to build an AF_XDP program that will send
 packets to userspace and if they are IPv6 ping packets reply.
 
-We will do using the automatically installed XDP program, but one of the
+We will do this using the automatically installed XDP program, but one of the
 assignments it to implement this manually.
```

Comment:
one of the _later_ assignments _is_ to ?

Reply:
It would be useful to describe what the default AF_XDP program does, i.e. redirects using an `XSKMAP` called `xsks_map` that is expected to be indexed by rx queue.